skimage/data/setup.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+from ._registry import legacy_datasets
```

Comment:
this line makes CI fail, please remove the relative import

Reply:
@emmanuelle do you have any idea why this fails, though?

Reply:
I think setup is run prior to install

Reply:
ah ok interesting, thanks @hmaarrfk. But removing the relative import also fails the CI... Should this be `from skimage.data._registry import legacy_datasets`?